examples/material_gallery/lib/demo/flexible_space_demo.dart

```diff
@@ -113,11 +113,11 @@ class FlexibleSpaceDemoState extends State<FlexibleSpaceDemo> {
               itemBuilder: (BuildContext context) => <PopupMenuItem<AppBarBehavior>>[
                 new PopupMenuItem<AppBarBehavior>(
                   value: AppBarBehavior.scroll,
-                  child: new Text('AppBar scrolls away')
+                  child: new Text('Toolbar scrolls away')
```

Comment:
The material spec uses the term "Toolbar", I was just making the demo match the spec. Happy to change it back if you want.